setup.cfg

```
@@ -19,6 +19,10 @@ install_requires =
 
 python_requires = >=3.6
 
+[options.entry_points]
+console_scripts =
+    heisenbridge = heisenbridge.__main__:main
```

You can save the `__main__.py` reindentation by doing this, I have found:

```suggestion
    heisenbridge = heisenbridge.__main__
```

---

oops, you can not do that after all. I'm not completely sure why my basic tests worked and the not-quite-basic ones now don't. Please disregard this suggestion!